{{cookiecutter.project_slug}}/Makefile

```diff
@@ -394,7 +397,7 @@ connect-train: _check_setup  ### Connect to the remote shell running on the trai
 jupyter: _check_setup upload-config upload-code upload-notebooks ### Run a job with Jupyter Notebook and open UI in the default browser
 	$(NEURO) run \
 		--name $(JUPYTER_JOB) \
-		--description "$(PROJECT_ID):jupyter" \
+		--tag "target:jupyter" --tag "jupyter-mode:$(JUPYTER_MODE)" $(_TAGS) \
```

Comment:
Get rid of `jupyter-mode:` tag (see https://docs.google.com/document/d/1Mv6uqnlvA3FdjW2KCp2YMwC_oYKWviCrlTX-f9G0-f4/edit?disco=AAAAJX0ioDk)